package-lock.json

```diff
@@ -0,0 +1,8896 @@
+          "resolved": "https://registry.npmjs.org/minimist/-/minimist-1.2.0.tgz",
+          "integrity": "sha1-o1AIsg9BOD7sH7kU9M1d95omQoQ=",
+          "dev": true
+        },
+        "path-exists": {
+          "version": "3.0.0",
+          "resolved": "https://registry.npmjs.org/path-exists/-/path-exists-3.0.0.tgz",
+          "integrity": "sha1-zg6+ql94yxiSXqfYENe1mwEP1RU=",
+          "dev": true
+        },
+        "sumchecker": {
+          "version": "2.0.2",
+          "resolved": "https://registry.npmjs.org/sumchecker/-/sumchecker-2.0.2.tgz",
+          "integrity": "sha1-D0LBDl0F2l1C7qPlbDOZo31sWz4=",
+          "dev": true,
+          "requires": {
+            "debug": "2.6.8"
+          }
+        },
+        "supports-color": {
+          "version": "4.2.1",
+          "resolved": "https://registry.npmjs.org/supports-color/-/supports-color-4.2.1.tgz",
+          "integrity": "sha512-qxzYsob3yv6U+xMzPrv170y8AwGP7i74g+pbixCfD6rgso8BscLT2qXIuz6TpOaiJZ3mFgT5O9lyT9nMU4LfaA==",
+          "dev": true,
+          "requires": {
+            "has-flag": "2.0.0"
+          }
+        }
+      }
+    },
+    "electron-builder-http": {
+      "version": "19.12.0",
+      "resolved": "https://registry.npmjs.org/electron-builder-http/-/electron-builder-http-19.12.0.tgz",
+      "integrity": "sha512-oJ84ygEMTKcddXlPuACvKRUY16aMJTvp7XxtbNfKX6+Uczg76ZUZoCvoHzhMrU1eSn2ZZmYzsIa/um21e5EnLQ==",
+      "dev": true,
+      "requires": {
+        "debug": "2.6.8",
+        "fs-extra-p": "4.4.0"
+      }
+    },
+    "electron-builder-util": {
+      "version": "19.12.0",
+      "resolved": "https://registry.npmjs.org/electron-builder-util/-/electron-builder-util-19.12.0.tgz",
+      "integrity": "sha512-i0sGc3YnruX6e4fZqcH4AIRzHzRpGffDNjJuCxu5copLNQoENTshWb+jFoFnQjdjePdppvfTfX6B4ESTIeOCoQ==",
+      "dev": true,
+      "requires": {
+        "7zip-bin": "2.1.0",
+        "bluebird-lst": "1.0.2",
+        "chalk": "2.0.1",
+        "debug": "2.6.8",
+        "electron-builder-http": "19.12.0",
+        "fcopy-pre-bundled": "0.3.4",
+        "fs-extra-p": "4.4.0",
+        "ini": "1.3.4",
+        "is-ci": "1.0.10",
+        "node-emoji": "1.8.1",
+        "source-map-support": "0.4.15",
+        "stat-mode": "0.2.2",
+        "tunnel-agent": "0.6.0"
+      },
+      "dependencies": {
+        "ansi-styles": {
+          "version": "3.2.0",
+          "resolved": "https://registry.npmjs.org/ansi-styles/-/ansi-styles-3.2.0.tgz",
+          "integrity": "sha512-NnSOmMEYtVR2JVMIGTzynRkkaxtiq1xnFBcdQD/DnNCYPoEPsVJhM98BDyaoNOQIi7p4okdi3E27eN7GQbsUug==",
+          "dev": true,
+          "requires": {
+            "color-convert": "1.9.0"
+          }
+        },
+        "chalk": {
+          "version": "2.0.1",
+          "resolved": "https://registry.npmjs.org/chalk/-/chalk-2.0.1.tgz",
+          "integrity": "sha512-Mp+FXEI+FrwY/XYV45b2YD3E8i3HwnEAoFcM0qlZzq/RZ9RwWitt2Y/c7cqRAz70U7hfekqx6qNYthuKFO6K0g==",
+          "dev": true,
+          "req
```

Comment:
I don't think you should introduce `package-lock.json` in an unrelevant PR. That should be the decision properly discussed with maintainer (probably in a new issue).